twitter-borrowbot/twitterbotErrors.py

```diff
@@ -0,0 +1,123 @@
+class Error(Exception):
+    pass
+
+class TweepyAuthenticationError(Error):
+    def __init__(self, error=None):
+        self.error = error
+
+    def __str__(self):
+        return "{0}: Failed to Authenticate with Twitter through Tweepy >> {1}".format(type(self).__name__, self.error)
+
+class FileIOError(Error):
+    def __init__(self, file=None, write=None, error="Error not provided"):
```

Comment:
Is `file` a file name, a file path, or a file pointer?  Explicit naming can help the caller.

`write` is a verb so the reader might assume that it is function.  What about `data` or `payload` instead?  

Python type hints on these parameters could also help the readers of this code.

Reply:
I will be sure to do this.